tests/test_identity_map_client.py

```diff
@@ -0,0 +1,172 @@
+import os
```

Comment:
I include all the test cases except [identityMapEmailsUseOwnHttp](https://github.com/IABTechLab/uid2-client-java/blob/main/src/test/java/com/uid2/client/IdentityMapIntegrationTests.java#L189) which I feel maybe not useful. Python has a short and clean code style. It is more like rewrite the identity map client call again. Happy to discuss if you feel it is useful to add it.

Reply:
That's right, own http is mainly for Java because there's no built-in async http.
Python users are unlikely to need that.